openjfx/samples/3DViewer/build.gradle

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2018, OpenJFX and/or its affiliates.
```

Comment:
You can probably lose the "and/or its affiliates" (not sure if it matters to you).

This brings up a point: to what organization does the Contributor Agreement for this project assign copyright? I guess whatever you choose should probably be here?